modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -296,9 +296,9 @@ export async function getObjectValidationRuleById<T>(
 	);
 }
 
-export async function getPickList(pickListId: number): Promise<ListTypeDefinition> {
+export async function getListTypeDefinition(listTypeDefinitionId: number): Promise<ListTypeDefinition> {
```

Comment:
@igor-franca check the sorting here too.

Reply:
It has already been done in the sort commit